package.json

```diff
@@ -24,7 +28,8 @@
     "micro": "^9.4.1",
     "micro-cors": "^0.1.1",
     "mongodb": "^6.1.0",
-    "undici": "^6.2.1"
+    "undici": "^6.2.1",
+    "zod": "3.22.4"
```

Comment:
Object validation library. Think the equivalent of yup or marshmallow if you're into Python.